composer.json

```diff
@@ -10,12 +10,12 @@
         }
     ],
     "require": {
-        "php":  ">=5.5"
+        "php":  "^7.1 || ^8.0"
```

Comment:
Why not just ">=7.1"? These days I'm seeing lots of libraries blocked out of php8 just because of the "^7.x" requirements, even though they would work perfectly fine on php8. 

If php9 came out soon, this library would not install when it would probably work out of the box.

We're using this library on PHP8 without any problems right now thanks to the ">=5.5" requirement, I'd suggest we keep the same ">=" requirement and not "^"

Reply:
https://github.com/thephpleague/tactician/commit/b8fda250aafcbfdcd4bb213c0397a853452ce3aa Addressed here :)